docs/components/table.md

```diff
@@ -483,6 +484,10 @@ Pass in an array of page sizes for the page size dropdown. If not provided will
 [15, 30, 50, 75, 100]
 ```
 
+### paginationType
+
+Pass in the type of pagination to be used. Options are `default` (page/pageSize) or `offset` (offset/pageSize)
```

Comment:
Can we show an example of what this looks like in the docs?

Reply:
Done 👍